tests/NRedisStack.Tests/Search/SearchTests.cs

```diff
@@ -702,7 +702,7 @@ public void AlterAdd()
         IDatabase db = redisFixture.Redis.GetDatabase();
         db.Execute("FLUSHALL");
         var ft = db.FT();
-        Schema sc = new Schema().AddTextField("title", 1.0);
+        Schema sc = new Schema().AddTextField("title", 1.0, sortable: true);
```

Comment:
I think we need to test both cases with sortable and not sortable text fields to ensure it's still usable for not sortable.